core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -362,6 +362,16 @@ public synchronized InetSocketAddress getAddress() throws UnavailableException {
     return mAddress;
   }
 
+  /**
+   * Gets the address from which to load the configuration for this client.
+   * <p>
+   * Checks whether {@link #mConfAddress} is null. If {@code mConfAddress}
+   * is null, the address should be found in {@link #mAddress}, and that is
+   * returned. Otherwise, {@code} mConfAddress is returned.
+   *
+   * @return the address to load configuration
```

Comment:
"the socket address to load configuration"

Reply:
Fixed.